system/Cache/Handlers/RedisHandler.php

```diff
@@ -118,6 +119,11 @@ public function initialize()
 			{
 				//              log_message('error', 'Cache: Redis authentication failed.');
 			}
+
+			if (isset($config['database']) && ! $this->redis->select($config['database']))
+			{
+				//              log_message('error', 'Cache: Redis select database failed.');
```

Comment:
This shouldn't be commented out.

Reply:
Perfect. Thanks! Merging.

Reply:
I had commented this out to log an error message.

But in the mean time, I notice only `redis->connect` will throw an RedisException is connection is failed.

`redis->auth` will only return a false rather an RedisException if auth is failed.

Should I enable the log for that either?

Reply:
Oh, yes, we should definitely be logging all of those situations. Though I've already merged this PR so you'll probably have to make another. Thanks!